### ISSUE TITLE
Syntax highlighting broken

### ISSUE THREAD

**Issue by arnarg** [arnarg/todotxt.nvim#10]
After the change in nui (see #7, fixed by #8) the syntax highlighting no longer works.

**Comment by xkapko** [comment]
hi! i'm still interested in contributing to this plugin and as i've discovered this plugin quite recently(yesterday) and i have not been able to take a look at most of the source code yet, i would like to ask you, what the expected behavior of the syntax highlighting is and what would you, as the plugin's author, consider to be the best way to move forward on both this issue and issue #9?

in the meantime, i will start by taking a look at `vim.ui.input` and i will try to figure out a way to integrate it into the plugin.

**Comment by xkapko** [comment]
Hi, a little status update: I've been able to get `vim.ui.input` to work with the prompt. There is still some cleanup that needs to be done but it mostly works now. I haven't been able to look at the syntax highlighting issue yet, but I will get to it soon enough. I won't really be able to communicate and make PRs for a couple of days now, as I have moved to a new city for university and sadly we don't have internet at the new place yet :(

I will get the `vim.ui.input`  code cleaned up and I will submit a PR as soon as possible.

**Comment by xkapko** [comment]
right, so i have submitted a pr (#12) which migrates the `Prompt` from `nui.input` to `vim.ui.input`. i have also began taking a look at how the highlighting is implemented so far and why it does not work. i have not clue how long it's gonna take me, but i think i could have it resolved & fixed in a few days. more updates to come!
